src/utils/suiPublishHotfix.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  getOriginalPackageId,
+  getPackageId,
+} from "@certusone/wormhole-sdk/lib/cjs/sui";
+import { SuiBuildOutput } from "@certusone/wormhole-sdk/lib/cjs/sui/types";
+import {
+  fromB64,
+  JsonRpcProvider,
+  normalizeSuiObjectId,
+  TransactionBlock,
+} from "@mysten/sui.js";
+
+export async function createWrappedOnSuiPrepare(
```

Comment:
Just for the record: what would the hotfix be for?

Reply:
added a comment